### PR TITLE
Data manager tests

### PR DESCRIPTION
**Pull request by akiskips**

### CODE REVIEW

File: apps/endpoint-registry/src/server.ts [hunk 1]
```diff
@@ -11,7 +11,11 @@ applyRoutes(healthRoute, router);
 applyMiddleware(middleware, router);
 applyRoutes(routes, router);
 
+<<<<<<< HEAD
```
Comment (kasimrehman) [inline apps/endpoint-registry/src/server.ts]: Merge Error!

File: apps/endpoint-registry/src/server.ts [hunk 2]
```diff
@@ -11,7 +11,11 @@ applyRoutes(healthRoute, router);
...
+const { ENDPOINT_REGISTRY_PORT = 4400 } = process.env;
+=======
```
Comment (kasimrehman) [inline apps/endpoint-registry/src/server.ts]: Merge Error!

File: apps/endpoint-registry/src/server.ts [hunk 3]
```diff
@@ -11,7 +11,11 @@ applyRoutes(healthRoute, router);
...
 const PORT = 4400;
+>>>>>>> master
```
Comment (kasimrehman) [inline apps/endpoint-registry/src/server.ts]: Merge Error!

File: apps/https-endpoint-ingress/src/server.ts [hunk 4]
```diff
@@ -66,11 +66,19 @@ if (
  * start the broker client and connect
  *  */
 
+<<<<<<< HEAD
```
Comment (kasimrehman) [inline apps/https-endpoint-ingress/src/server.ts]: Merge Error!

File: apps/https-endpoint-ingress/src/server.ts [hunk 5]
```diff
@@ -66,11 +66,19 @@ if (
...
+const { HTTPS_ENDPOINT_INGRESS_PORT = 2000 } = process.env;
+
+router.listen(HTTPS_ENDPOINT_INGRESS_PORT, () => {
+  logger.info(
+    `A Server is running http://localhost:${HTTPS_ENDPOINT_INGRESS_PORT}...`
+=======
```
Comment (kasimrehman) [inline apps/https-endpoint-ingress/src/server.ts]: Merge Error!

File: apps/https-endpoint-ingress/src/server.ts [hunk 6]
```diff
@@ -66,11 +66,19 @@ if (
...
 const PORT = 2000;
 
 router.listen(PORT, () => {
   logger.info(
     `A Server is running http://localhost:${PORT}...`
+>>>>>>> master
```
Comment (kasimrehman) [inline apps/https-endpoint-ingress/src/server.ts]: Merge Error!

File: apps/storage-adapter-mongodb/src/server.ts [hunk 7]
```diff
@@ -11,12 +11,21 @@ applyMiddleware(middleware, router);
 applyRoutes(routes, router);
 applyMiddleware(errors, router);
 
+<<<<<<< HEAD
```
Comment (kasimrehman) [inline apps/storage-adapter-mongodb/src/server.ts]: Merge Error!

File: apps/storage-adapter-mongodb/src/server.ts [hunk 8]
```diff
@@ -11,12 +11,21 @@ applyMiddleware(middleware, router);
...
+const { STORAGE_ADAPTER_MONGODB_PORT = 3100 } = process.env;
+//const server = http.createServer(router);
+
+router.listen(STORAGE_ADAPTER_MONGODB_PORT, () => {
+  console.log(
+    `A Server is running http://localhost:${STORAGE_ADAPTER_MONGODB_PORT}...`
+=======
```
Comment (kasimrehman) [inline apps/storage-adapter-mongodb/src/server.ts]: Merge Error!

File: apps/storage-adapter-mongodb/src/server.ts [hunk 9]
```diff
@@ -11,12 +11,21 @@ applyMiddleware(middleware, router);
...
 const PORT = 3100;
 //const server = http.createServer(router);
 
 router.listen(PORT, () => {
   console.log(
     `A Server is running http://localhost:${PORT}...`
+>>>>>>> master
```
Comment (kasimrehman) [inline apps/storage-adapter-mongodb/src/server.ts]: Merge Error!

File: docker-compose.yml [hunk 10]
```diff
@@ -26,8 +26,11 @@ services:
     env_file:
       - .compose-envs/adapter-registry.env
       - .compose-envs/data-manager.env
+<<<<<<< HEAD
```
Comment (kasimrehman) [inline docker-compose.yml]: Merge Error!

File: docker-compose.yml [hunk 11]
```diff
@@ -26,8 +26,11 @@ services:
...
+=======
```
Comment (kasimrehman) [inline docker-compose.yml]: Merge Error!

File: docker-compose.yml [hunk 12]
```diff
@@ -26,8 +26,11 @@ services:
...
     ports:
       - "4000:4000"
+>>>>>>> master
```
Comment (kasimrehman) [inline docker-compose.yml]: Merge Error!

File: k8s/onboarding-skill/i40aas-cluster.yaml [hunk 13]
```diff
@@ -63,6 +63,22 @@ spec:
...
+                configMapKeyRef:
+                  name: service-addresses
+                  key: REGISTRY_URL
+            - name: DATA_MANAGER_URL
+              valueFrom:
+                configMapKeyRef:
+                  name: service-addresses
+                  key: DATA_MANAGER_URL
+=======
+>>>>>>> master
```
Comment (kasimrehman) [inline k8s/onboarding-skill/i40aas-cluster.yaml]: Merge Error!

**Comment by kasimrehman** [comment]
Please fix the merge errors.

File: apps/data-manager/src/server.ts [hunk 14]
```diff
@@ -58,18 +64,22 @@ if (
   ADAPTER_REGISTRY_ADMIN_PASSWORD &&
   ADAPTER_REGISTRY_URL_SUFFIX
 ) {
+
+let getAdaptersULR = new URL(buildUrl(ADAPTER_REGISTRY_PROTOCOL,ADAPTER_REGISTRY_HOST,ADAPTER_REGISTRY_PORT,ADAPTER_REGISTRY_URL_SUFFIX));
```
[the remark this answers is not in the document]
Comment (akiskips) [reply apps/data-manager/src/server.ts]: Right, ill change it

**Pull request merged**